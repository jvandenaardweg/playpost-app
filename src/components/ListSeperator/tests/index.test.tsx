import React from 'react';
import { render, RenderAPI } from 'react-native-testing-library';
import { ListSeperator } from '../index';

describe('ListSeperator', () => {

  describe('rendering', () => {
    let wrapper: RenderAPI;

    beforeAll(() => {
      wrapper = render(<ListSeperator />);
    });

    it('should render correctly', () => {
      expect(wrapper.toJSON()).toMatchSnapshot();
    });
  });
});
