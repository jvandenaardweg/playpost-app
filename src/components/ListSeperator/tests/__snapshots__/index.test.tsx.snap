// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListSeperator rendering should render correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ffffff",
      "paddingLeft": 12,
      "paddingRight": 12,
    }
  }
>
  <View
    style={
      Object {
        "borderBottomColor": "#e5e5e5",
        "borderBottomWidth": 0.5,
      }
    }
  />
</View>
`;
